Description: Disable checkstyle and codenarc tasks during build
 Gradle failed to build on amd64 due to an apparent bug in upstream.
 As extra comment, the issue is not present when gradle is build with
 Oracle JDK.
Bug-Debian: http://bugs.debian.org/719415
Bug: http://issues.gradle.org/browse/GRADLE-2858
Last-Update: 2013-08-11
Forwarded: not-needed
--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -16,8 +16,8 @@
 import org.apache.ivy.plugins.resolver.FileSystemResolver
 
 apply plugin: 'groovy'
-apply plugin: 'checkstyle'
-apply plugin: 'codenarc'
+//apply plugin: 'checkstyle'
+//apply plugin: 'codenarc'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
 
@@ -48,5 +48,5 @@
     compile "org.gradle.jarjar:jarjar:1.2.1"
 }
 apply from: '../gradle/compile.gradle'
-apply from: '../gradle/codeQuality.gradle'
+//apply from: '../gradle/codeQuality.gradle'
 apply from: '../gradle/classycle.gradle'
